production-relstorage.cfg

```diff
@@ -0,0 +1,35 @@
+# Addition to the productiion.cfg for deployments which uses relstorage.
+# Drops filestorage and zeo part
+
+# Following variables has to be defined in the instance0 section
+#  - relstorage-type
+#  - relstorage-user
+#  - relstorage-pw
+#  - relstorage-shared-blob-dir
+#  - relstorage-commit-lock-id
+#  - (relstorage-blob-cache-size)
+
+
+[buildout]
+parts -=
+    filestorage
+    zeo
+
+[supervisor]
+programs =
+    ${supervisor:program-instances}
+
+[instance0]
+zeo-client = off
+zeo-address =
+
+relstorage-blob-cache-size = 1gb
+
+rel-storage =
+    type ${:relstorage-type}
```

Comment:
+1 for having all important variables in the buildout section (prefixed with `relstorage-`).
Hint: Please do not reference other sections from the default values since this can cause problems.